custom_components/ha-google-home/const.py

```diff
@@ -67,7 +69,7 @@
 
 # Access token only lives about 1 hour
 # Update often to fetch timers in timely manner
-UPDATE_INTERVAL = 15
+UPDATE_INTERVAL = 60 * 5  # every 5 minutes
```

Comment:
This is for how often the sensor will update. 

You are right, I m just afraid 15 seconds is way too often. I have about 4 devices home and it takes at least 20-30 per request-response cycle. I think it's jamming quite a bit.

That's why I think we need a custom setting for people to adjust it themselves #58. It's a bit tricky with small-timers like 2-3 minutes because they indeed can fall into that 5 mins gap to not be updated right away. But since we provide `local_time` when the timer/alarm goes off the fronend of HA should take care of countdowns :)

Reply:
That's scary. I have 10 devices and I don't think it depends on number of them, it's still around 20 seconds. I think it's better to investigate where this time is gone, because all queries are pretty fast, probably it's zeroconf. Then probably we don't need to run zeroconf for updates.

Reply:
Also we don't need to retrieve keys every time and queries to different devices can run in parallel. So it should be something closer to 100ms than 20s.

Reply:
Parallel queries are handled automatically, right? 🤔  HA is taking care of that i believe
A bit confused, what do you mean by not needing to retrieve keys?

Reply:
Yeh, you are actually right. Can you please add it as an issue, so we optimize it? :)

Reply:
#60

Reply:
Let's just keep 5mins as a default refresh rate now. 
We have a ticket to implement setting for customization and performance improvement ticket now. Superb!

Reply:
I use my google home as a kitchen timer, and this will not work for that scenario. Also for me it only takes about 5-10 secs before i retrieves alarms/timers.(But i also only have one device) And the frontend card i have made uses firetime to know when to fire and does the counting in the frontend.

Please either revert this change or make it something like 1 min or so until we have found a better way to handle how often we fetch timers 😊

Reply:
That's reasonable 👍  I'll set it to 1 minute for now and then we'll see if we can introduce a better config setting so each can tune it accordingly 💯